pkgs/development/python-modules/pillow/default.nix

```diff
@@ -24,6 +24,16 @@ import ./generic.nix (rec {
     hash = "sha256-nIK1s+BDx68NlXktDSDM9o9hof7Gs1MOcYtohCJyc5Y=";
   };
 
+  patches = [
+    # Pull in zlib-1.3 fix pending upstream inclusion
+    #   https://github.com/python-pillow/Pillow/pull/7344
+    (fetchpatch {
+      name = "zlip-1.3.patch";
```

Comment:
Typo zlip

Reply:
Good catch! Renamed to `name = "zlib-1.3.patch";`.